examples/CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# pytket examples
+# Contributing new notebooks
 
 See the pytket examples built with jupyterbook [here](https://tket.quantinuum.com/examples).
```

Comment:
I think this paragraph ("See the pytket examples...") isn't needed here - you could put it in a separate examples/README file, but I don't think we need it now this is CONTRIBUTING.md.

Reply:
Ah forgot to do that. Thanks! I've deleted the extra lines.